client/src/main/com/sinch/sdk/domains/verification/models/webhooks/VerificationResponseCalloutSpeech.java

```diff
@@ -0,0 +1,78 @@
+package com.sinch.sdk.domains.verification.models.webhooks;
+
+/** Verification response details related to call out speech */
+public class VerificationResponseCalloutSpeech {
```

Comment:
You're right, I missed this point.
This should be highlighted from OAS file too.
I'll work to refine specs to this direction and will use it for the PR related to new generator version migration.
For this reason: I'll change to destination for this PR no longer being onto onto 1.1-next branch(in case of...) but to a feature branch: too much layers/PRs required now for a single destination